pkg/storage/snapshot/snapshot.go

```diff
@@ -630,6 +630,8 @@ func (ctrl *VMSnapshotController) getSnapshotPVC(namespace, volumeName string) (
 		return pvc, nil
 	}
 
+	log.Log.Warningf("No VolumeSnapshotClass for %s", pvc.Spec.storageClassName)
```

Comment:
typo capital S -> StorageClassName

Reply:
Thanks for the pickup, it seems that the value also needed to be dereferenced.